src/pytorch_lightning/loggers/neptune.py

```diff
@@ -272,13 +269,10 @@ def __init__(
         agg_default_func: Optional[Callable[[Sequence[float]], float]] = None,
         **neptune_run_kwargs: Any,
     ):
+        if not _NEPTUNE_AVAILABLE:
+            raise ModuleNotFoundError(str(_NEPTUNE_AVAILABLE))
         # verify if user passed proper init arguments
         self._verify_input_arguments(api_key, project, name, run, neptune_run_kwargs)
-        if neptune is None:
-            raise ModuleNotFoundError(
-                "You want to use the `Neptune` logger which is not installed yet, install it with"
-                " `pip install neptune-client`."
```

Comment:
Why are we not showing this anymore? We want to tell the user how to install it.

Reply:
The package requirement is included in the message, for the error in the linked issue:

```python
ModuleNotFoundError: Requirement 'neptune-client' not met, DistributionNotFound: The 'webcolors; extra == "format"' distribution was not found and is required by jsonschema
```

Note that not mentioning pip is more generic, as this would be different for conda users

Reply:
Discussed in Slack, this can be added in the `RequirementCache` implementation